ng-ncc-app/src/app/common/services/notes.service.ts

```diff
@@ -407,7 +407,7 @@ export class NotesService {
         if (additional_notes) {
             note.push(additional_notes);
         }
-        return note.join('\n');
+        return note.join(';');
```

Comment:
We might want a space here to make everything more readable.

Reply:
okay. I'll add a space